alibi/explainers/tests/test_permutation_importance.py

```diff
@@ -0,0 +1,360 @@
+import pytest
```

Comment:
Should we have more "functional" tests like in the PDVariance case, e.g. checking that for synthetic problem an uninformative feature gets a ~1/0 score for ratio/difference and an informative features gets a non-zero score?

Also thinking if we should have a convergence test checking whether the estimate method is close to the exact method with some number of repeats.

Reply:
I think the following test are sufficient as they are sanity checks on some synthetic datasets:
* [test_compute_exact](https://github.com/RobertSamoilescu/alibi/blob/3dbcdd523229638fdef27f767f945c2d4250e165/alibi/explainers/tests/test_permutation_importance.py#L241)
* [test_compute_estimate](https://github.com/RobertSamoilescu/alibi/blob/3dbcdd523229638fdef27f767f945c2d4250e165/alibi/explainers/tests/test_permutation_importance.py#L287)
* [test_explain_exact](https://github.com/RobertSamoilescu/alibi/blob/3dbcdd523229638fdef27f767f945c2d4250e165/alibi/explainers/tests/test_permutation_importance.py#L332)

I tried to avoid using some randomly generate datasets because of the randomness in the results. Same thing for convergence. Tried to avoid introducing randomness since some tests may occasionally fail if the consistency threshold is too small. I haven't tried, but I remember we had some issues with other explainers. Of course, one can argue that for a large sample size this is very unlikely to happen ....

Let me know if you still consider that we need more tests.